### DISCUSSION TITLE
Do you all have a process around PRs?

### DISCUSSION THREAD

**Discussion by cdt-eth** [nounsDAO/nouns-monorepo#406] [Ideas/Contribute]
Do you all have a process around PRs?

**Comment by solimander** [comment]
Please discuss any changes in #developers in [discord.gg/nouns](https://discord.gg/nouns) prior to contributing to reduce duplication of effort and in case there is any prior work that may be useful to you.

Take a look at [open issues](https://github.com/nounsDAO/nouns-monorepo/issues) if looking for features and/or bugs to implement or fix.

For the sake of the reviewer, one PR per feature or bug fix is preferred.
